css/src/mixins/chevron.scss

```diff
@@ -0,0 +1,33 @@
+$chevron-color: $text !default;
+$chevron-arrow-size: 0.525em !default;
+$chevron-arrow-border-width: 1px !default;
+$chevron-arrow-actual-width: 0.75em !default;
+$chevron-down-rotate: rotate(-45deg) !default;
+$chevron-right-rotate: rotate(-135deg) !default;
+$chevron-up-rotate: rotate(-225deg) !default;
+$chevron-up-rotate-scale-xy: rotate(-45deg) scaleY(-1) scaleX(-1);
+
+%chevron {
+	width: $chevron-arrow-size;
+	height: $chevron-arrow-size;
+	transform-origin: center;
+	border: $chevron-arrow-border-width solid $chevron-color;
+	border-block-start: 0;
+	border-inline-end: 0;
+	pointer-events: none;
+	content: ' ';
+	z-index: $zindex-active;
+	inset-inline-end: $control-padding-horizontal;
+}
+
+@mixin chevron-right {
```

Comment:
Do we need to account for rtl?

Reply:
Yea we should, I've added rtl styles. To test add `dir="rtl"` to any parent element or details element

Reply:
We don't have to change the direction in doc's UI because it's an up and down docon that aligns right or left. Since we've removed the docon dependency and are using rotations this needs to be accounted for in rtl for this component.